ui/nuxeo-routing-behavior.js

```diff
@@ -138,7 +138,8 @@ export const RoutingBehavior = {
       if (obj.path && obj.uid) {
         entityType = 'document';
       } else {
-        throw new Error(`cannot resolve route: object does not have an "entity-type"`);
+        console.warn(`cannot resolve route: object "${obj}" does not have an "entity-type"`);
+        return '';
```

Comment:
Is there a particular advantage in returning `''` here?
```suggestion
        return;
```

Reply:
yes IIRC when I tried with only return I got also an error in other place `UNDEFINED` and as the method should return a string if I understand correctly I did with an empty String.

thx